src/app.test.js

```diff
@@ -180,16 +202,32 @@ test("shows outstanding todos when filter is set to Outstanding", () => {
   expect(queryByDisplayValue(todoList, "Take dog for a walk!")).toBeVisible();
 });
 
-/**
- * SETUP
- * render todo app
- * input user name
- * add few todos
- * destroy todo app
- * ACT
- * re render todo app
- * input same user name
- * ASSERT
- * see previously added todos rendered
- */
-test.todo("renders todo list for existing user");
+test("renders empty todo list when changing to a nonexistent user", () => {
```

Comment:
I believe there is no such a thing as a "user" atm, only on a theoretical level

what we've got is a bunch of todo items stored in `localStorage` where the key is generated from the "username", maybe "nonexistent user" would be better to phrase e.g. as "new user" here?